style.css

```
@@ -1441,44 +1259,94 @@ dd.accordion-content {
   padding: 1em;
 }
 
-.icon-wrap {
+/*--------------------------------------------------
```

Stylelint (17.2.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)